test/structure/test_metal_atom.py

```diff
@@ -47,37 +41,38 @@ def test_get_donor_atoms():
     assert len(metalcenter1.get_donor_atoms()) == 4
     assert len(metalcenter2.get_donor_atoms()) == 4
 
+
 def test_get_donor_mapper():
-    stru = PDBParser().get_structure(f"{DATA_DIR}1NVG.pdb")
+    stru = PDBParser.get_structure(f"{DATA_DIR}1NVG.pdb")
     metalcenter1: MetalUnit = stru.metalcenters[0]
     donor_mapper = metalcenter1.get_donor_mapper()
     assert len(donor_mapper) == 4
     assert list(donor_mapper.values())[0][0].name == "OE2"
 
-@pytest.mark.TODO
+
 def test_get_radii_method_good_input():
     """Ensuring the MetalUnit.get_radii() method works correctly."""
     pdb_file = f"{DATA_DIR}/1NVG.pdb"
-    metals: List[MetalUnit] = structure_from_pdb(pdb_file).metals
+    metals: List[MetalUnit] = PDBParser.get_structure(pdb_file).metals
     assert np.isclose(metals[0].get_radii(), 0.88)
     assert np.isclose(metals[0].get_radii("vdw"), 1.39)
 
     assert np.isclose(metals[0].get_radii(), metals[1].get_radii())
     assert np.isclose(metals[0].get_radii("vdw"), metals[1].get_radii("vdw"))
 
-@pytest.mark.TODO
+
 def test_residue_to_metal():
     """Ensuring the residue_to_metal() method works."""
-    residue = Residue("A.ZN.1", [Atom(line_idx=1, atom_name="Zn")])
+    residue = Residue(5, "Zn", list())
     metal: MetalUnit = residue_to_metal(residue)
     assert isinstance(metal, MetalUnit)
-    assert metal.atom_name_ == "Zn"
+    assert metal.name == "Zn"
```

Comment:
In this test metal is a residue object and normally in PDB naming format. It should be ZN instead of Zn. That is also the reason we ahve the method atom_name.
This reminds me of emphasizing the atom/residue naming problem (check out the docstring of Atom class). The problem for now is, especially for atom names, it is strongly coupled with the PDB format. Each atom name in its nature represents a specific topology in the residue and a unique name is a perfect way to represent that. So the solution I think of is just to keep the atom name consistant and convert everything potentially in the future to this one defined with PDB. e.g.: in the case we support a neo-PDB we will need to create a map to convert back and from PDB atom names